src/test/unit/merge-compare.test.js

```diff
@@ -233,6 +233,38 @@ describe('mergeProfilesForDiffing function', function () {
     expect(mergedProfileThreadB.nativeSymbols.libIndex).toEqual([1, 1]);
     expect(mergedThread.nativeSymbols.libIndex).toEqual([0, 0, 1, 1]);
   });
+
+  it('should use marker timing if there are no samples', () => {
+    const profiles = [
+      getProfileWithMarkers([
+        ['Thread1 Marker1', 2],
+        ['Thread1 Marker2', 3, 5],
+        ['Thread1 Marker3', 6, 7],
+      ]),
+
+      getProfileWithMarkers([
+        ['Thread1 Marker1', 5],
+        ['Thread1 Marker2', 6, 8],
+        ['Thread1 Marker3', 10, 15],
+      ]),
+    ];
+
+    const profileState = stateFromLocation({
+      pathname: '/public/fakehash1/',
+      search: '?thread=0&v=3',
+      hash: '',
+    });
+
+    const { profile } = mergeProfilesForDiffing(profiles, [
+      profileState,
+      profileState,
+    ]);
+
+    expect(getTimeRangeIncludingAllThreads(profile)).toEqual({
+      end: 21,
+      start: 4,
```

Comment:
as you see, the start is `4` that is `2+2` (twice the smallest start), but it should be `0` because the diffing code should have moved all timestamps to be alignes on `0`.